website/content/specification/syntax/constraints.md

```diff
@@ -163,6 +163,21 @@ And the following document.
 
 The expect constraint would pass for each `sibling` in the `parent` named "p1", and would fail for each `sibling` in the `parent` named "p2".
 
+## `expect` constraints
+
+The `expect` constraint is a type of Metaschema constraint that restricts field or flag value(s) based on the evaluation of a test Metapath expression.
+
+The `@target` attribute of an `<expect>` constraint specifies the node(s) in a document instance whose value is restricted by the constraint.
+
+The `@test` attribute of an `<expect>` constraint specifies the logical condition to be evaluated against each value node resulting from evaluating the `@target`. This expression must evaluate to [a Metaschema boolean value](/specification/datatypes#boolean) `true` or `false`.
+
+When the `@test` expression evaluates to `true` for a value node, then the value node is considered valid and passes the constraint.
+
+When the `@test` expression evaluates to `false` for a value node, then the value node is considered not valid and fails the constraint.
+
+A constraint may have an optional [`@level`](#level) attribute and/or an optional child [`<message>`](#message) element to indicate severity and documentation explaining how the target nodes are invalid.
```

Comment:
```suggestion
A constraint may have an OPTIONAL [`@level`](#level) attribute and/or an OPTIONAL child [`<message>`](#message) element to indicate severity and documentation explaining how the target nodes are invalid.
```

Reply:
I am embarrassed I did not know that OPTIONAL was one of the RFC keywords, today I learn. 🤦

Reply:
Sigh, I missed clicking add to batch on this one. I will pull locally and touch it up, sorry!

Reply:
I am not sure why it is not getting picked up but amending the feedback commit to https://github.com/usnistgov/metaschema/commit/71c046c8a6af453731fc513f1775f604d4bd74f7 should have closed this, resolving.